classes/class-wc-connect-shipping-label.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Shipping_Label' ) ) {
+
+	class WC_Connect_Shipping_Label {
+
+		private $settings_store;
+
+		public function __construct( WC_Connect_Service_Settings_Store $settings_store ) {
+			$this->settings_store = $settings_store;
+		}
+
+		protected function get_form_data( WC_Order $order ) {
+			$formData = [];
+			$contents = array();
+
+			foreach( $order->get_items() as $item ) {
+				$product = $order->get_product_from_item( $item );
+				$height = 0;
+				$length = 0;
+				$weight = $product->get_weight();
+				$width = 0;
+
+				if ( $product->has_dimensions() ) {
+					$height = $product->get_height();
+					$length = $product->get_length();
+					$width  = $product->get_width();
+				}
+
+				$contents[] = array(
+					'height' => $height,
+					'product_id' => $item['product_id'],
+					'length' => $length,
+					'quantity' => $item['qty'],
+					'weight' => $weight,
+					'width' => $width
+				);
+			}
+			$formData[ 'cart' ] = $contents;
+
+			foreach( $order->get_address( 'shipping' ) as $key => $value ) {
+				$formData[ 'orig_' . $key ] = '';
```

Comment:
The origin address will need to be stored somewhere, for sure. I just don't know where yet. After it's introduced for the first time, the flow will just automatically skip past the "Origin address" step.

Because we need the origin address, right?